src/EditPlayerModal.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import {connect} from 'react-redux';
+import {
+  savePlayer
+} from './actions/playerActions';
+import ActionType from './actions/ActionType'
+import PropTypes from "prop-types";
+
+class EditPlayerModal extends React.Component {
+
+    constructor(props) {
+        super(props)
+        this.state = {
+        }
+        this.updatePlayerName = this.updatePlayerName.bind(this)
+        this.updateSlackName = this.updateSlackName.bind(this)
+        this.saveChanges = this.saveChanges.bind(this)
+        this.closeModal = this.closeModal.bind(this)
+    }
+
+    updatePlayerName(e) {
+        this.setState({ playerName: e.target.value })
+    }
+
+    updateSlackName(e) {
+        this.setState({ slackName: e.target.value })
+    }
+
+    saveChanges() {
+        this.props.savePlayer(this.props.editingPlayer.playerId, this.state.playerName, this.state.slackName)
+    }
+
+    closeModal() {
+        this.props.closeModal()
+    }
+
+    render(){
+        if (this.props.editingPlayer !== undefined)
```

Comment:
Recommend replacing the `if` block with a inline `if` with the `&&` operator: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator